linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml

```diff
@@ -11,5 +11,3 @@ description: |-
     In Red Hat Enterprise Linux servers and workstations, hardware token login
     {{% endif %}}
     is not enabled by default and must be enabled in the system settings.
-
-platform: machine
```

Comment:
The group smart_card_login conatins rule service_pcscd_enabled, which uses systemd check but OpenSCAP cannot evaluate systemd checks in containers.

Reply:
Agreed. Rule reverted to as machine only.

Reply:
> The group smart_card_login conatins rule service_pcscd_enabled, which uses systemd check but OpenSCAP cannot evaluate systemd checks in containers.

This is something that needs to be fixed, btw.